### PR TITLE
Fixes #18615: Embed augeas in the agent 7.0 and later

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/18615

### CODE REVIEW

File: rudder-agent/SOURCES/Makefile.in [hunk 1]
```diff
@@ -31,7 +31,7 @@ zlib_RELEASE = 1.2.11
 zlib_SHA256 = c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
 # Original URL: http://ftp.fr.debian.org/debian/pool/main/l/lmdb/lmdb_$(LMDB_RELEASE).orig.tar.xz
 lmdb_RELEASE = 0.9.24
-lmdb_SHA256 = 8e7200fdc9bb09b0cfad1d4eee8b20d539e43fe05c7ef9406247d64659c4a480
```
Comment (amousset) [inline rudder-agent/SOURCES/Makefile.in:34]: why does this change?

Comment (peckpeck) [reply rudder-agent/SOURCES/Makefile.in:34]: because it was forgotten when dependencies were upgraded, that why 7.0 dosen't build since then

**Comment by peckpeck** [comment]
some things are missing, do not merge yet

File: rudder-agent/SOURCES/Makefile.in (hunk 1 above)
Comment (amousset) [reply rudder-agent/SOURCES/Makefile.in:34]: Ok :+1:

**Review by amousset — APPROVED** [review]

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**